controller/callbackcontroller.php

```diff
@@ -430,6 +433,8 @@ public function track($doc, $users, $key, $status, $url, $token) {
                     }
 
                     $newData = $documentService->Request($url);
+
+                    $this->logger->debug("Track put content " . $file->getPath(), array("app" => $this->appName));
```

Comment:
after few seconds of closing the editor, I saw the debug msg acknowledging write to original file. Also checked on other account after restart of onlyoffice.